core/workspace_svg.js

```diff
@@ -2328,7 +2331,7 @@ WorkspaceSvg.prototype.getScale = function() {
  * @package
  */
 WorkspaceSvg.prototype.scroll = function(x, y) {
-  Blockly.hideChaff(/* opt_onlyClosePopups */ true);
+  this.hideChaff(/* opt_onlyClosePopups */ true);
```

Comment:
[The style guide recommends](https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments):

```suggestion
  this.hideChaff(/* opt_onlyClosePopups= */ true);
```

Reply:
Done!